From ltsi-dev-bounces@lists.linuxfoundation.org Mon May 28 02:47:12 2012
From: "Shimoda, Yoshihiro" <yoshihiro.shimoda.uh@renesas.com>
Date: Mon, 28 May 2012 18:47:07 +0900
Subject: [LTSI-dev] [PATCH 06/21] sh: move CLKDEV_xxx_ID macro to sh_clk.h
To: ltsi-dev@lists.linuxfoundation.org
Message-ID: <4FC3499B.9050300@renesas.com>


>From 540cdd813dd77275e44ec24701364e1895cd98bd Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Wed, 6 Jul 2011 02:54:11 +0000
Subject: sh: move CLKDEV_xxx_ID macro to sh_clk.h

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Paul Mundt <lethal@linux-sh.org>
(cherry picked from commit 1522043bf73ef0e9e61dc512a0f3cdbec1cbf89f)

Conflicts:

	arch/sh/kernel/cpu/sh4a/clock-sh7343.c
	arch/sh/kernel/cpu/sh4a/clock-sh7366.c
	arch/sh/kernel/cpu/sh4a/clock-sh7722.c
	arch/sh/kernel/cpu/sh4a/clock-sh7723.c
	arch/sh/kernel/cpu/sh4a/clock-sh7724.c
	arch/sh/kernel/cpu/sh4a/clock-sh7785.c
	arch/sh/kernel/cpu/sh4a/clock-sh7786.c
	arch/sh/kernel/cpu/sh4a/clock-shx3.c

Signed-off-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
---
 arch/sh/kernel/cpu/sh4a/clock-sh7757.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/arch/sh/kernel/cpu/sh4a/clock-sh7757.c b/arch/sh/kernel/cpu/sh4a/clock-sh7757.c
index 86bf188..3b097b0 100644
--- a/arch/sh/kernel/cpu/sh4a/clock-sh7757.c
+++ b/arch/sh/kernel/cpu/sh4a/clock-sh7757.c
@@ -101,9 +101,6 @@ static struct clk mstp_clks[MSTP_NR] = {
 	[MSTP220] = SH_CLK_MSTP32(&div4_clks[DIV4_P], MSTPCR2, 20, 0),
 };

-#define CLKDEV_CON_ID(_id, _clk) { .con_id = _id, .clk = _clk }
-#define CLKDEV_ICK_ID(_cid, _did, _clk) { .con_id = _cid, .dev_id = _did, .clk = _clk }
-
 static struct clk_lookup lookups[] = {
 	/* main clocks */
 	CLKDEV_CON_ID("extal", &extal_clk),
-- 
1.7.1
_______________________________________________
LTSI-dev mailing list
LTSI-dev@lists.linuxfoundation.org
https://lists.linuxfoundation.org/mailman/listinfo/ltsi-dev

